src/main/scala/com/amazon/deequ/suggestions/ConstraintSuggestionRunBuilder.scala

```diff
@@ -220,6 +236,15 @@ class ConstraintSuggestionRunBuilder(val data: DataFrame) {
 
     List[Any](testsetRatio, testsetSplitRandomSeed)
   }
+
+  // implement this wrapper to not violate scalastyle requirement on argcount
+  private def kllWrapper(
+    kllParameters: Option[KLLParameters],
+    predefinedColumnDataTypes: Map[String, DataTypeInstances.Value])
+  : List[Any] = {
+
+    List[Any](kllParameters, predefinedColumnDataTypes)
```

Comment:
return ` (Option[KLLParameters], Map[String, DataTypeInstances.Value])` instead of  `List[Any]`

Reply:
Okay. Changing testWrapper return type to (Option[Double], Option[Long]) as well.